priv/static/swagger.json

```diff
@@ -1688,7 +1688,7 @@
         ]
       }
     },
-    "/api/session-occurence": {
+    "/api/session-occurrence": {
```

Comment:
why are we making changes in swagger.json file. It's randomly generated no?

Reply:
I have reverted those changes. Please take a look.